tls/s2n_security_policies.h

```diff
@@ -193,6 +193,5 @@ int s2n_validate_kem_preferences(const struct s2n_kem_preferences *kem_preferenc
 S2N_RESULT s2n_validate_certificate_signature_preferences(const struct s2n_signature_preferences *s2n_certificate_signature_preferences);
 S2N_RESULT s2n_security_policy_get_version(const struct s2n_security_policy *security_policy,
         const char **version);
-S2N_RESULT s2n_signature_preferences_validate_supported(
-        const struct s2n_signature_preferences *cert_sig_preferences,
-        const struct s2n_cert_info *info);
+S2N_RESULT s2n_security_policy_cert_signature_scheme_validate(
+        const struct s2n_security_policy *security_policy, const struct s2n_cert_info *info);
```

Comment:
Now I'm really nitting: since it IS the security policy doing the validating now, you can go back to s2n_security_policy_validate_sig_scheme_supported :) Or I like s2n_security_policy_validate_cert_signature